rules/integrations/google_workspace/defense_evasion_application_removed_from_blocklist_in_google_workspace.toml

```
@@ -26,7 +26,50 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "Application Removed from Blocklist in Google Workspace"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Application Removed from Blocklist in Google Workspace
+
+Google Workspace Marketplace is an online store for free and paid web applications that work with Google Workspace services and third-party software. Listed applications are based on Google APIs or Google Apps Script and created by both Google and third-party developers.
+
+Marketplace applications require access to specific Google Workspace resources. Individual users with the appropriate permissions can install applications in their Google Workspace domain. Administrators have additional permissions that allow them to install applications for an entire Google Workspace domain. Consent screens typically display permissions and privileges the user needs to install an application. As a result, malicious Marketplace applications may require more permissions than necessary or have malicious intent.
+
+Google clearly states that they are not responsible for any Marketplace product that originates from a source that isn't Google.
+
+This rule identifies a Marketplace blocklist update that consists of Google Workspace account with administrative privileges manually removing a previously blocked application. 
+
+#### Possible investigation steps
+
+- Identify the associated user accounts by reviewing `user.name` or `user.email` fields in the alert.
+- This rule relies on data from `google_workspace.admin`, thus indicating the associated user has administrative privileges to the Marketplace.
+- With access to the Google Workspace admin console, visit the `Security > Investigation` tool with filters for the user email and event is `Assign Role` or `Update Role` to determine if new cloud roles were recently updated.
+- With the user account, review other potentially related events within the last 48 hours.
```

Suggestion based on your feedback [here](https://github.com/elastic/detection-rules/pull/2540#discussion_r1159797798):

```suggestion
- After identifying the involved user account, review other potentially related events within the last 48 hours.
```